test.py

```diff
@@ -11,10 +11,8 @@ def check_call(cmd):
   subprocess.check_call(cmd.split(' '))
 
 def checked_call_with_output(cmd, expected, stderr=None):
-  proc = subprocess.Popen(cmd.split(' '), stdout=subprocess.PIPE, stderr=stderr)
-  stdout, stderr = proc.communicate()
-  assert not proc.returncode, 'call must have succeeded'
-  assert expected in (stdout + stderr), 'call did not have the right output: ' + stdout + stderr
+  stdout = subprocess.check_output(cmd.split(' '), stderr=stderr)
```

Comment:
another pedantic thing I just noticed. `cmd.split(' ')` is not quite *exactly* the same as `cmd.split()`, but does this code actually care about the difference?

Reply:
for now this code assumes simple commands with just a single space as a separator, but I guess if we want more general whitespace to be split on then we could use `split()`?